lib/govuk_index/publishing_event_worker.rb

```diff
@@ -6,6 +6,8 @@ class NotFoundError < StandardError; end
   class UnknownDocumentTypeError < StandardError; end
   class ValidationError < StandardError; end
 
+  DOCUMENT_TYPES_WITHOUT_BASE_PATH = %w(contact world_location).freeze
```

Comment:
This duplicates knowledge from the content schemas. Maybe the validation here should be swapped for a schema validation?

Reply:
I agree with @dwhenry here.

These are valid content items but anything without a base path (or some other way of generating a link) should be excluded from search.

Reply:
Yeah I agree, I probably meant it should be done by checking the presence of a `base_path`, so you don't duplicate information in the schema.

Reply:
(but don't let this hold you off from merging, I'm just a tourist on this repo)